src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Issue22035.cs

```diff
@@ -14,7 +14,8 @@ public Issue22035(TestDevice device) : base(device)
 		public override string Issue => "[Android] CarouselView: VirtualView cannot be null here, when clearing and adding items on second navigation";
 
 		[Test]
-		[FailsOnMacWhenRunningOnXamarinUITest("This test is failing, App.Back() is not working as expected.")]
+		[FailsOnIOSWhenRunningOnXamarinUITest("This test is failing, likely due to product issue. More information: https://github.com/dotnet/maui/issues/22287")]
```

Comment:
Don't use these attributes to remove tests. 

Use the if/defines and then add a note there

```C#
TEST_FAILS_ON_IOS;
TEST_FAILS_ON_CATALYST;
TEST_FAILS_ON_WINDOWS
```

Reply:
@PureWeen  In the commit below, all test fail attributes were changed to be similar to "FailsOnIOSWhenRunningOnXamarinUITest".  Which attribute should we use to remove the future test cases? 

https://github.com/dotnet/maui/commit/8e603a4f36cf24300dc360a7e8e2f7c7e926e167#diff-4568e04585599619b86b4cd254e55d6520986835c27364968b941fb5b176292e
<img width="706" alt="image" src="https://github.com/user-attachments/assets/04d8b393-4d36-4da5-bae5-4876add2a074">